improver_tests/regrid/test_grid.py

```diff
@@ -213,3 +214,29 @@ def test_create_regrid_cube():
     assert cube_v.coord("forecast_reference_time") == source_cube_latlon.coord(
         "forecast_reference_time"
     )
+    assert cube_v.standard_name == "air_temperature"
+    assert cube_v.long_name is None
+
+
+def test_create_regrid_cube_non_standard_name():
+    """Test the create_regrid_cube function with a non-standard name"""
+
+    source_cube_latlon = set_up_variable_cube(
+        np.ones((2, 5, 5), dtype=np.float32),
+        name="not_a_standard_name",
+        time=datetime(2018, 11, 10, 8, 0),
+        frt=datetime(2018, 11, 10, 0, 0),
+    )
+    target_cube_equalarea = set_up_variable_cube(
+        np.ones((10, 10), dtype=np.float32), spatial_grid="equalarea"
+    )
+    data = np.repeat(1.0, 200).reshape(2, 10, 10)
+    cube_v = create_regrid_cube(data, source_cube_latlon, target_cube_equalarea)
+    assert cube_v.shape == (2, 10, 10)
+    assert cube_v.coord(axis="x").standard_name == "projection_x_coordinate"
+    assert cube_v.coord(axis="y").standard_name == "projection_y_coordinate"
+    assert cube_v.coord("forecast_reference_time") == source_cube_latlon.coord(
+        "forecast_reference_time"
+    )
+    assert cube_v.standard_name is None
+    assert cube_v.long_name == "not_a_standard_name"
```

Comment:
Can we have a test for a cube in which the `var_name` is different to the standard name and check that these are both present on the regridded cube.

Reply:
Done.